packages/web3torrent/.env.development

```diff
@@ -0,0 +1,7 @@
+FIREBASE_PROJECT='rock-paper-scissors-dev-4ff8d'
+FIREBASE_API_KEY='AIzaSyAOvhDzJir_El3O6SJ2xQlrpOisnObq6zw'
+
+GANACHE_CACHE_FOLDER=../../.ganache-deployments
+GANACHE_PORT=8547
+USE_GANACHE_DEPLOYMENT_CACHE=true
+DEPLOYER_ACCOUNT_INDEX=2
```

Comment:
This probably doesn't belong here.

Reply:
How is everyone else setting the shared ganache details? I'm confused as to why the project doesn't have a `.env.development` already checked in?

Reply:
It does a `.env.development` checked in already.